packages/lib/storybook/stories/components/PayTo.stories.tsx

```diff
@@ -2,23 +2,135 @@ import { Meta, StoryObj } from '@storybook/preact';
 import { PaymentMethodStoryProps } from '../types';
 import { ComponentContainer } from '../ComponentContainer';
 import { Checkout } from '../Checkout';
-import PayTo, { PayToConfiguration } from '../../../src/components/PayTo/PayTo';
+import PayTo, { MandateType, PayToConfiguration } from '../../../src/components/PayTo/PayTo';
+import { http, HttpResponse } from 'msw';
 
-type PayToStory = StoryObj<PaymentMethodStoryProps<PayToConfiguration>>;
+// extend the default story args so we can change mandate top level
+interface ExtendedStoryArgs extends PaymentMethodStoryProps<PayToConfiguration> {
+    mandate: MandateType;
+}
+type PayToStory = StoryObj<ExtendedStoryArgs>;
+
+const MANDATE_EXAMPLE = {
+    amount: '25900', // [Mandatory] for PayTo - Mandate Amount field
+    amountRule: 'exact', // [Mandatory] for PayTo - Needs to be Localised
+    endsAt: '2025-12-31', // [Mandatory] for PayTo - Date format
+    frequency: 'adhoc', // [Mandatory] for PayTo - Needs to be Localised
+    remarks: 'testThroughFlow1', // [Mandatory] for PayTo - Needs to be Localised as "Description"
+    count: '3' // [Optional] will be returned only if the merchant sends it
+    //startsAt: '2025-02-01' // [Optional] will be returned only if the merchant sends it
+};
 
 const meta: Meta<PaymentMethodStoryProps<PayToStory>> = {
     title: 'Components/PayTo'
 };
 
+// const payToPaymentObject = {
```

Comment:
Do we need these code comments about the `payToPaymentObject` and `paymentResponse` or can they be removed? (not sure if you left them because dev/mock purposes)

Reply:
I will clean up in the last commit, serves a bit of documentation.